book/src/integrations/pam_and_nsswitch.md

```diff
@@ -489,3 +489,10 @@ may have your laptop in a park without wifi.
 Clearing the cache, however, completely wipes all local data about all accounts and groups. If you
 are relying on this cached (but invalid) data, you may lose access to your accounts until other
 communication issues have been resolved.
+
+### Home directories are not created via SSH
+
+Ensure that `UsePAM yes` is set in `sshd_config`. Without this the pam session module won't be
+triggered which prevents the background task being completed.
+
+
```

Comment:
The extra newlines here are making deno sad

Reply:
I have no idea how they were added, just deleted them.

Reply:
yeah, easy to do :smile